Description: Skip CoincidentRootLoci test that runs out of memory on i386
Bug: https://github.com/Macaulay2/M2/issues/1581
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-11-05

--- a/M2/Macaulay2/packages/CoincidentRootLoci/tests.m2
+++ b/M2/Macaulay2/packages/CoincidentRootLoci/tests.m2
@@ -185,7 +185,6 @@
 assert(sort apply(apply(subsets X10,singularLocus),L -> sort apply(L,partition)) === sort apply(A10,sort))
 X11 = coincidentRootLocus(toList(11:1),ZZ/101,Variable=>x)
 A11 = {{},{{11}},{{11}},{{10,1}},{{11}},{{8,3},{9,2},{10,1}},{{9,1,1}},{{11}},{{7,4},{8,3},{10,1}},{{9,2}},{{7,2,2},{7,3,1},{8,2,1},{9,1,1}},{{8,1,1,1}},{{11}},{{6,5},{7,4},{10,1}},{{6,5},{8,3},{9,2}},{{6,4,1},{7,3,1},{9,1,1}},{{6,3,2},{7,2,2},{8,2,1}},{{6,2,2,1},{6,3,1,1},{7,2,1,1},{8,1,1,1}},{{7,1,1,1,1}},{{6,5}},{{6,5},{7,4},{9,2}},{{5,5,1},{6,4,1},{9,1,1}},{{8,3}},{{5,3,3},{5,4,2},{5,5,1},{6,3,2},{7,3,1},{8,2,1}},{{5,3,3},{5,4,1,1},{6,3,1,1},{8,1,1,1}},{{7,2,2}},{{5,2,2,2},{5,3,2,1},{6,2,2,1},{7,2,1,1}},{{5,2,2,1,1},{5,3,1,1,1},{6,2,1,1,1},{7,1,1,1,1}},{{5,5,1},{6,1,1,1,1,1}},{{7,4}},{{4,4,3},{5,4,2},{6,4,1}},{{5,4,1,1}},{{4,4,3},{5,3,3},{7,3,1}},{{4,4,3},{5,4,2},{6,3,2},{7,2,2}},{{4,3,2,2},{4,3,3,1},{4,4,2,1},{5,3,2,1},{5,4,1,1},{6,3,1,1},{7,2,1,1}},{{4,3,3,1},{4,4,1,1,1},{4,4,3},{5,3,1,1,1},{7,1,1,1,1}},{{4,3,2,2},{4,4,2,1},{5,2,2,2},{6,2,2,1}},{{4,2,2,2,1},{4,3,2,1,1},{4,4,1,1,1},{5,2,2,1,1},{6,2,1,1,1}},{{4,2,2,1,1,1},{4,3,1,1,1,1},{4,4,2,1},{5,2,1,1,1,1},{6,1,1,1,1,1}},{{4,4,1,1,1},{5,1,1,1,1,1,1}},{{5,3,3}},{{4,3,3,1}},{{3,3,3,2},{4,3,2,2},{5,3,2,1}},{{3,3,2,2,1},{3,3,3,1,1},{3,3,3,2},{4,3,2,1,1},{5,3,1,1,1}},{{3,3,3,1,1},{4,3,1,1,1,1}},{{5,2,2,2}},{{3,2,2,2,2},{3,3,2,2,1},{4,2,2,2,1},{5,2,2,1,1}},{{3,2,2,2,1,1},{3,3,2,1,1,1},{3,3,2,2,1},{4,2,2,1,1,1},{4,4,3},{5,2,1,1,1,1}},{{3,2,2,1,1,1,1},{3,3,1,1,1,1,1},{3,3,2,1,1,1},{4,2,1,1,1,1,1},{5,1,1,1,1,1,1}},{{3,3,1,1,1,1,1},{4,1,1,1,1,1,1,1}},{{3,2,2,2,2}},{{2,2,2,2,2,1},{3,2,2,2,1,1}},{{2,2,2,2,1,1,1},{3,2,2,1,1,1,1},{4,4,2,1}},{{2,2,2,1,1,1,1,1},{3,2,1,1,1,1,1,1},{4,4,1,1,1}},{{2,2,1,1,1,1,1,1,1},{3,1,1,1,1,1,1,1,1}},{}}
-assert(sort apply(apply(subsets X11,singularLocus),L -> sort apply(L,partition)) === sort apply(A11,sort))
 ///
 
 TEST ///
